packages/nvidia_nat_langchain/src/nat/plugins/langchain/tools/exa_internet_search.py

```
@@ -0,0 +1,99 @@
+                    livecrawl=tool_config.livecrawl,
+                    text={"max_characters": 3000},
+                )
+                if not search_response.results:
+                    return f"No web search results found for: {question}"
+                # Format - SearchResponse.results contains Result objects with .url and .text attrs
+                web_search_results = "\n\n---\n\n".join([
+                    f'<Document href="{doc.url}"/>\n{doc.text}\n</Document>'
+                    for doc in search_response.results if doc.text
+                ])
+                return web_search_results or f"No web search results found for: {question}"
+            except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)